lumicks/pylake/population/dwelltime.py

```diff
@@ -198,11 +234,30 @@ def __init__(
             max_observation_time,
             options=self._optim_options,
         )
-        self.bootstrap = DwelltimeBootstrap()
+        self._bootstrap = DwelltimeBootstrap(self)
```

Comment:
Add the same TODO here as below. If you go with the earlier suggestion, this would become:
```suggestion
        # TODO: remove with deprecation
        self._bootstrap = DwelltimeBootstrap(self, np.empty((n_components, 0)), np.empty((n_components, 0)))
```